react/features/base/dialog/components/AbstractDialog.js

```diff
@@ -56,7 +56,12 @@ export default class AbstractDialog extends Component {
         /**
          * Key to use for showing a title.
          */
-        titleKey: React.PropTypes.string
+        titleKey: React.PropTypes.string,
+
+        /**
+         * String to use as a title instead of a titleKey.
```

Comment:
Please expand the comment to clarify that specifying a truthy `titleString` value will take precedence over `titleKey`.